src/cascadia/TerminalApp/KeyChordSerialization.cpp

```diff
@@ -240,12 +251,18 @@ winrt::hstring KeyChordSerialization::ToString(const KeyChord& chord)
     }
     else
     {
-        if (vkeyNamePairs.find(vkey) != vkeyNamePairs.end())
+        bool foundKey = false;
+        for (const auto& pair : vkeyNamePairs)

```

Comment:
Why not keep the `find` thing from before?

Reply:
Since the key, value pairs were flipped, I flipped the logic to minimize the amount of reference changes. I swapped the if statement with and the for loop in the `FromString` and `ToString` functions.